backend/src/services/eagleEyeContentService.ts

```diff
@@ -86,10 +87,22 @@ export default class EagleEyeContentService {
     // We do not want what we have already accepted or rejected
     const filters = await this.findNotInbox()
 
-    const lambdaArn =
-      getConfig().NODE_ENV === 'production'
-        ? 'arn:aws:lambda:eu-central-1:359905442998:function:EagleEye-prod-search'
-        : 'arn:aws:lambda:eu-central-1:359905442998:function:EagleEye-staging-search'
+    if (KUBE_MODE) {
+      try {
+        const response = await request
+          .post(API_CONFIG.premiumApiUrl)
+          .send({ queries: keywords, nDays, filters })
+
+        return response.body
+      } catch (error) {
+        console.log('error while calling eagle eye server!', error)
+        throw new Error400('en', 'errors.wrongEagleEyeSearch.message')
+      }
+    }
+
+    const lambdaArn = IS_PROD_ENV
```

Comment:
Suggestion:
Add a comment `// TODO-kube` with everything that will be deleted once we ditch Serverless. Not needed if you are already writing it somewhere else.

Reply:
I'll try to do my best - I think it's gonna be quite a lot though.